package-lock.json

```diff
@@ -0,0 +1,16308 @@
+{
```

Comment:
This is generated by npm, you have also a `yarn.lock`, you need to choose one and keep one.